tests/memcpy_threads/memcpy_threads.c

```diff
@@ -5,17 +5,20 @@
 #include <string.h>
 #include <time.h>
 #include "libminiasync.h"
+#include "core/os.h"
 
 int
 test_threads_memcpy_multiple_same_char(unsigned memcpy_count,
```

Comment:
this function and the one below are almost identical. Try to abstract away the boilerplate.